tests/ember_debug/profile-manager-test.js

```diff
@@ -1,8 +1,138 @@
+import setupEmberDebugTest from '../helpers/setup-ember-debug-test';
 
-test('Ember Debug - Construction', function (assert) {
-  let manager = new ProfileManager();
-  assert.ok(!!manager, 'it was created');
-  assert.equal(manager.profiles.length, 0, 'it has no profiles');
+module('Ember Debug - profile manager component highlight', function (hooks) {
+  setupEmberDebugTest(hooks, {
+    routes() {
+      this.route('simple');
+    },
+  });
+  hooks.beforeEach(async function () {
+    EmberDebug.IGNORE_DEPRECATIONS = true;
+
+    this.owner.register(
+      'route:simple',
+      EmberRoute.extend({
+        model() {
+          return EmberObject.create({
+            toString() {
+              return 'Simple Model';
+            },
+          });
+        },
+      })
+    );
+
+    this.owner.register(
+      'controller:simple',
+      Controller.extend({
+        toString() {
+          return 'App.SimpleController';
+        },
+      })
+    );
+
+    this.owner.register(
+      'component:test-foo',
+      EmberComponent.extend({
+        classNames: ['simple-component'],
+        toString() {
+          return 'App.TestFooComponent';
+        },
+      })
+    );
+
+    this.owner.register(
+      'component:test-bar',
+      EmberComponent.extend({
+        tagName: '',
+        toString() {
+          return 'App.TestBarComponent';
+        },
+      })
+    );
+
+    /*
+    Setting line-height to normal because normalize.css sets the
+    html line-height to 1.15. This seems to cause a measurement
+    error with getBoundingClientRect
+    */
+    this.owner.register(
+      'template:simple',
+      hbs('Simple {{test-foo}} {{test-bar}}', {
+        moduleName: 'my-app/templates/simple.hbs',
+      })
+    );
+    this.owner.register(
+      'template:components/test-foo',
+      hbs('test-foo', {
+        moduleName: 'my-app/templates/components/test-foo.hbs',
+      })
+    );
+    this.owner.register(
+      'template:components/test-bar',
+      hbs(
+        '<!-- before --><div class="another-component"><span>test</span> <span class="bar-inner">bar</span></div><!-- after -->',
+        { moduleName: 'my-app/templates/components/test-bar.hbs' }
+      )
+    );
+  });
+
+  test('ProfileManager can be created', function (assert) {
+    const profileManager = new ProfileManager();
+
+    assert.ok(!!profileManager, 'it was created');
+    assert.equal(profileManager.profiles.length, 0, 'it has no profiles');
+  });
+
+  test('Highlights for render show and destroy as expected', async function (assert) {
+    await visit('/simple');
+    const profileManager = new ProfileManager();
+    const simpleComponent = find('.simple-component');
+    const bounds = {
+      parent: {
+        childNodes: {},
+      },
+      first: {
+        node: simpleComponent,
+      },
+      last: {
+        node: simpleComponent,
+      },
+    };
+    const payload = {
+      view: {
+        [Symbol('BOUNDS')]: bounds,
```

Comment:
could we get real payload here (to be sure we match exsting implemetation once changed)

Reply:
great idea, done.